pkg/exec/exec.go

```diff
@@ -96,8 +119,56 @@ func RunCommandInLabeledPod(ctx context.Context, clientsets *k8sutil.Clientsets,
 	return stdout.String()
 }
 
-// execCmdInPod exec command on specific pod and wait the command's output.
-func execCmdInPod(ctx context.Context, clientsets *k8sutil.Clientsets, command, podName, containerName, podNamespace, clusterNamespace string, args []string, stdout, stderr io.Writer) error {
+// execCmdInPodWithoutReturn exec command on specific pod and print the command's output directly on shel.
+func execCmdInPodWithoutReturn(ctx context.Context, clientsets *k8sutil.Clientsets, command, podName, containerName, podNamespace, clusterNamespace string, args []string) {
```

Comment:
This method seems mostly duplicated of the original method. What if this method took the `existOnError` parameter? Then it could just change the behavior needed instead of duplicating the method. That might help with my confusion on the other comment as well.

Reply:
Merge both in one method and also added one small comment on what `exitOnError` condition doing.